### ISSUE TITLE
node server on wlan0

### ISSUE THREAD

**Issue by humblewizard** [joyent/node#15516]
How do you set up a server on wlan0 ? 
hostapd is set up with dhcp and wifi clients can connect and be given an ipaddress. How do I get my node http server to respond to traffic on wlan0. Couldn't find examples of that, only server connections to localhost.

**Comment by bahamat** [comment]
Looks like you already asked that question here: http://stackoverflow.com/q/29640163/818112

And you were directed to the answer here: http://stackoverflow.com/a/20996813/818112

**Comment by humblewizard** [comment]
OK thanks I'm half way there but this nob is still confused.

On my server->$ifconfig  the wan0 shows up with address 192.168.150.1

On the server node script I set:

       http.createServer(function(){}).listen(80,"192.168.150.1");

So now from any of the WiFi clients and from the browser on the server I can put into the browser 192.168.150.1 and I get the pages that the nodejs server can fetch. However when the browser has any other url like "index.html" or "google.com" routes through to the internet present on eth0. I would like all or at least a few specific urls requests on the wlan0 to hit the nodejs server. 

Maybe this is not a nodejs issue but a routing table issue which I know even less about.

On the server if I were to add to the hosts table the domain names I worried about and directed them to 192.168.150.1 would that work. I'm trying that now.

This may sound nefarious but it's not. I'm using a KOBO e-ink reader as a display board and before it will enable it's WiFi it want's to check in with a specific external URL. In use I will not have a network on eth0 and I want to make the reader happy by serving a page for the\at external URL.

Thanks.
